test/adapters/dio_adapter_test.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:dio/dio.dart';
+import 'package:http_mock_adapter/http_mock_adapter.dart';
+import 'package:test/test.dart';
+
+void main() {
+  Dio dio;
+  DioAdapter dioAdapter;
+
+  Response<dynamic> response;
+
+  const path = 'https://example.com';
+
+  const data = {
+    'payload': {
+      'data': 'content',
+    }
+  };
+
+  const statusCode = 200;
```

Comment:
removed extra breaklines. they are relevant.

```suggestion
  const path = 'https://example.com';
  const data = {
    'payload': {
      'data': 'content',
    }
  };
  const statusCode = 200;
```